src/pages/App.tsx

```diff
@@ -185,6 +185,18 @@ export default function App() {
     [account]
   )
 
+  // redirect address to landing pages until implemented
+  const shouldRedirectToAppInstall = pathname?.startsWith('/address/')
```

Comment:
how do users get to `/address`? are we routing them from the mobile app?

Reply:
yes the share button from the mobile app profile screen